src/module/vue/components/character-sheet-tabs/sf-connections.vue

```diff
@@ -33,15 +33,16 @@
 </style>
 
 <script setup lang="ts">
-import { computed, inject, Ref } from 'vue'
+import { computed, inject, provide, Ref } from 'vue'
 import { $ActorKey } from '../../provisions'
 import OrderButtons from '../order-buttons.vue'
-import ProgressBox from '../progress/progress-box.vue'
+import ProgressListItem from '../progress/progress-list-item.vue'
 import BtnFaicon from '../buttons/btn-faicon.vue'
 import { ProgressDataProperties } from '../../../item/itemtypes'
 
 const actor = inject('actor') as Ref
 const $actor = inject($ActorKey)
+provide($ActorKey, $actor)
```

Comment:
No need for this, if you're `inject`ing it, that means it's already been `provide`d by some higher node in the tree.

```suggestion
```

Reply:
oh, so it doesn't have to be passed down manually? good to know!

Reply:
Yes: `provide` holds up a value to the current node and all child nodes, any of which can `inject` using the same key. Depth doesn't matter. Check out [this article](https://vuejs.org/guide/components/provide-inject.html) to get a better explanation, and remember that if something is on an "actor" sheet, that sheet should be providing `$ActorKey` (done by the [base class](https://github.com/ben/foundry-ironsworn/blob/eabf1cb9f25cabc888e7568b0c027fee66567092/src/module/vue/vueactorsheet.ts#L20)) and any kind of sheet that's editing an item will do the same (done by the [base class](https://github.com/ben/foundry-ironsworn/blob/eabf1cb9f25cabc888e7568b0c027fee66567092/src/module/vue/vueitemsheet.ts#L22)).